js/main.js

```diff
@@ -1,2 +1,47 @@
 /* global data */
 /* exported data */
+
+var inputElement = document.querySelector('#url');
+// console.log(inputElement);
+
+var imgSource = document.querySelector('img');
+
+function input(event) {
+  // console.log(inputElement.value);
+  imgSource.setAttribute('src', inputElement.value);
+}
+
+inputElement.addEventListener('input', input);
+
+var formElement = document.querySelector('form');
+
+var titleEl = document.querySelector('#title');
+var urlEl = document.querySelector('#url');
+var notesEl = document.querySelector('#notes');
+
+var entryNumber = data.nextEntryId;
+
+var inputValue = {
+  title: '',
+  url: '',
+  notes: '',
+  ID: entryNumber
+};
```

Comment:
You should move this variable inside the `submit` event handler function. You can also just assign values to it directly, the properties don't need an initial value of an empty string.